src/Twig/RecordExtension.php

```diff
@@ -31,16 +32,20 @@ class RecordExtension extends AbstractExtension
     private $translator;
 
     /** @var string */
-    private $menu = null;
+    private $sidebarMenu = null;
 
     /** @var TaxonomyRepository */
     private $taxonomyRepository;
 
-    public function __construct(MenuBuilder $menuBuilder, TranslatorInterface $translator, TaxonomyRepository $taxonomyRepository)
+    /** @var FrontendMenuBuilder */
+    private $frontendMenuBuilder;
+
+    public function __construct(MenuBuilder $menuBuilder, FrontendMenuBuilder $frontendMenuBuilder, TranslatorInterface $translator, TaxonomyRepository $taxonomyRepository)
```

Comment:
this class does too much. can you extract menu related functions to MenuExtension?

Reply:
I had some other ideas as well.. The taxonomy suff should be split off as well. Will do so in a separate PR, to prevent rebase-hell.